services/comments/comments.go

```diff
@@ -11,10 +11,41 @@ import (
 
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/git"
+	"code.gitea.io/gitea/modules/log"
 	"code.gitea.io/gitea/modules/setting"
+	api "code.gitea.io/gitea/modules/structs"
 	"code.gitea.io/gitea/services/gitdiff"
 )
 
+// CreateIssueComment creates a plain issue comment.
+func CreateIssueComment(doer *models.User, repo *models.Repository, issue *models.Issue, content string, attachments []string) (*models.Comment, error) {
+	comment, err := models.CreateComment(&models.CreateCommentOptions{
+		Type:        models.CommentTypeComment,
+		Doer:        doer,
+		Repo:        repo,
+		Issue:       issue,
+		Content:     content,
+		Attachments: attachments,
+	})
+	if err != nil {
+		return nil, fmt.Errorf("CreateComment: %v", err)
```

Comment:
In other your moved out code there was just return err, so we should probably keep consistency here also
```suggestion
		return nil, err
```

Reply:
done.